example/spec/parallel/testAPI/sqsRuleSpec.js

```diff
@@ -238,5 +241,14 @@ describe('The SQS rule', () => {
     it('messages are picked up and removed from source queue', async () => {
       await expectAsync(waitForQueueMessageCount(queues.sourceQueueUrl, 0)).toBeResolved();
     });
+
+    it('stores incoming messages on S3', async () => {
```

Comment:
So we can only test that the message exists on S3 in the case of an unprocessable message, otherwise it would have been deleted from S3 possibly before this test could check for it?

Reply:
Should we possible clean up this message manually then in the `afterAll` for this block?

Reply:
Oh, cleaning up the message afterwards is a good point. Yes, we can only check that it exists if it is unprocessable.

Reply:
[`5223038` (#2132)](https://github.com/nasa/cumulus/pull/2132/commits/522303829adf7b5c63fec589dced18ebcf084cf4)